ci/build-test-qemu.yaml

```diff
@@ -0,0 +1,37 @@
+job:
+  strict: true
+recipe:
+  git_ref: master
+  git_url: https://github.com/openshift/os
```

Comment:
For PRs to work, we need to inherit these bits from Prow's `src` container build right?

Reply:
Yeah, this is POC for now. That _should_ come later.